helmfile.d/snippets/defaults.gotmpl

```diff
@@ -193,16 +193,27 @@ environments:
                 egressPublic: true
                 ingressPrivate: true
             resourceQuota:
-                - name: pods
-                  value: "50"
                 - name: services.loadbalancers
                   value: "0"
+                - name: services.nodeports
+                  value: "0"
+                - name: limits.cpu
+                  value: "24"
+                - name: requests.cpu
+                  value: "24"
+                - name: limits.memory
+                  value: "32Gi"
+                - name: requests.memory
+                  value: "32Gi"
+                - name: count/pods
```

Comment:
maybe we should drop `count/` for `pods` to stay consistent with the other count namespaced resources, see: https://kubernetes.io/docs/concepts/policy/resource-quotas/

Reply:
makes sense/ Thanks.